packages/@aws-cdk/pipelines/README.md

```diff
@@ -144,6 +144,21 @@ new MyPipelineStack(this, 'PipelineStack', {
 });
 ```
 
+Deployment is done by default with `CodePipeline` engine by first generating
+change set, and then executing it. This allow to add e.g. approval before 
+actual deployment happens. On the other hand this prevents to retry the 
+deployment, as change set can't be re-generated if execution of it fails for
+any reason. Deployment can be done as single step by setting 
+`CodePipelineProps.prepareStep` to false (for whole pipeline). It's also 
+possible to change default functionality per stage (with 
+`StageOptions.prepareStep`) or per stack in stage (with
+`StageOptions.prepareStepForStacks`). See examples from tests:
+
+- [whole pipeline](./test/integ.pipeline-without-prepare.ts)
+- [single stage](./test/integ.pipeline-stage-without-prepare.ts)
+- [single stack in stage](./test/integ.pipeline-stage-stack-without-prepare.ts)
```

Comment:
These relative hyperlinks will only work on GitHub. 

* They do not work here: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.pipelines-readme.html
* They do not work here: https://www.npmjs.com/package/@aws-cdk/pipelines

Instead, put in a code example directly that showcases how to use this feature in a simple way. It's okay to not cover every permutation.